spec/fixtures/iex/client/access_denied.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: https://cloud.iexapis.com/v1//stock/msft/quote?token=test-iex-api-publishable-token
+    uri: https://cloud.iexapis.com/v1//stock/msft/quote
```

Comment:
double check that this was intended, I think the test needs to pass a token through